src/pages/ResetPassword/components/ResetForm/ResetForm.tsx

```diff
@@ -21,7 +21,13 @@ import {
 } from './ResetForm.styles';
 import { resetSchema } from './ResetForm.schema';
 
-export const ResetForm = ({ setEmail }: { setEmail: Dispatch<SetStateAction<string>> }) => {
+export const ResetForm = ({
+  setEmail,
+  onSubmitForTest,
+}: {
+  setEmail?: Dispatch<SetStateAction<string>>;
+  onSubmitForTest?: () => void;
+}) => {
```

Comment:
Could you please transfer TS types definition to the ResetForm.props.ts or ResetForm.types.ts?